nbs/03_data.core.ipynb

```diff
@@ -674,6 +679,29 @@
     "test_eq(next(iter(dls2.valid)),tensor([1,1,1,1]))"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(Pipeline: _TestTfm, Pipeline: _TestTfm)"
+      ]
+     },
+     "execution_count": null,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "dls3 = DataLoaders.from_dsets(start, start, bs=4, val_bs=8, after_item=[_TestTfm()], val_after_batch=[_TestTfm()])\n",
+    "test_eq(len(dls3.train.after_batch.fs),0)\n",
+    "test_eq(len(dls3.valid.after_batch.fs),1)\n",
+    "dls.train.after_item, dls.valid.after_item"
```

Comment:
personally I think its better for this line to be a test:  
```
test_eq(len(dls.train.after_item.fs), len(dls.valid.after_item.fs))
```

Additionally, shall we also test `bs` is set properly on train & valid?  
```
test_eq(dls.train.bs, 4)
test_eq(dls.valid.bs, 8)
```